app/views/users/edit.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1>Edit The User</h1>
+
+
+
+<%= form_with model: @user do |form| %>
+
+    <%=form.label :email, "Enter email" %>
+    <%=form.text_field :email, placeholder: "Enter email" %>
+    <%@user.errors.full_messages_for(:email) do |message| %>
+    <div> <%=message%>< /div>
+    <%end%>
+
+    <%=form.label :username %>
+    <%=form.text_field :username %>
+    <%@user.errors.full_messages_for(:username) do |message| %>
+        <div> <%=message%>< /div>
+        <%end%>
+
+    <%=form.label :password %>
+    <%=form.text_field :password %>
+    <%@user.errors.full_messages_for(:password) do |message| %>
+        <div> <%=message%>< /div>
+        <%end%>
+    <%=form.submit %>
+<%end%>
```

Comment:
1. In line 8 instead of `text_field` use `email_tag`
2. In line 20 instead of `text_field` use `password_field`